ui/lib/core/addon/components/page/error.ts

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import Component from '@glimmer/component';
+import { service } from '@ember/service';
+import { tracked } from '@glimmer/tracking';
+import { ResponseError } from '@hashicorp/vault-client-typescript';
+
```

Comment:
Could we add docs here?

Reply:
By docs do you mean jsdoc annotations? If so I'm curious if they are used to generate documentation or for anything else other than commenting on component args. If the latter is the case than it seems redundant given that the `Args` are typed here.

Reply:
There is a docify thing that generates component docs for us but it's more about what the component does (e.g. the description is the most important part imo).